package.json

```diff
@@ -169,67 +167,56 @@
     "build-server": "npm run -s env -- webpack --display-error-details --config webpack.config.node.js",
-    "build-client": "npm run -s env -- node --max_old_space_size=8192 ./node_modules/webpack/bin/webpack.js --display-error-details --config webpack.config.js",
-    "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
-    "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
-    "clean": "npm run -s clean:build && npm run -s clean:devdocs && npm run -s clean:public",
-    "clean:build": "npm run -s rm -- build && npm run -s rm -- server/bundler .json && npm run -s rm -- .babel-cache",
-    "clean:devdocs": "npm run -s rm -- server/devdocs/search-index.js && npm run -s rm -- server/devdocs/proptypes-index.json && npm run -s rm -- server/devdocs/components-usage-stats.json",
-    "clean:public": "npm run -s rm -- public .css .css.map .js .js.map && npm run -s rm -- public/sections .css .css.map && npm run -s rm -- public/sections-rtl .css .css.map",
+    "clean": "make clean",
     "codemod": "node bin/codemods/run",
     "distclean": "npm run -s clean && npm run -s rm -- node_modules",
     "docker": "docker run -it --name wp-calypso --rm -p 80:3000 wp-calypso",
     "env": "cross-env-shell NODE_PATH=$NODE_PATH:server:client:.",
     "eslint-branch": "node bin/eslint-branch.js",
-    "install-if-deps-outdated": "node bin/install-if-deps-outdated.js",
-    "install-if-no-packages": "node bin/install-if-no-packages.js",
-    "lint": "run-s -s lint:*",
-    "lint:config-defaults": "node server/config/validate-config-keys.js",
-    "lint:css": "stylelint \"client/**/*.scss\" --syntax scss",
-    "lint:js": "npm run -s install-if-deps-outdated && eslint-eslines .",
-    "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
-    "precommit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
-    "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
-    "rm": "node bin/rm.js",
+    "lint": "make lint",
+    "postbuild-desktop": "npm run -s bundle",
+    "postcss": "postcss -r -u postcss-custom-properties -u autoprefixer -c postcss.config.json",
     "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
-    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && node bin/version-check.js",
-    "start": "npm run -s build",
     "poststart": "npm run -s env -- cross-env-shell node $NODE_ARGS build/bundle.js",
-    "reformat-files": "node bin/reformat-files.js",
-    "pretest": "npm run -s install-if-deps-outdated",
-    "test": "run-s -s test-client test-server",
+    "preanalyze-bundles": "cross-env-shell CALYPSO_ENV=production npm run -s build-client -- --profile --json > stats.json",
```

Comment:
The `preTask` and `postTask` scripts should be near their respective task IMHO. Alphabetical order hurts more than helps in this situation.

Reply:
we can restore the sort later. it was needlessly indirect for me while trying to port the scripts

also, are these run by `npm-s`?

Reply:
Yes, `npm-s a b` acts just like `npm run a && npm run b`, that means `prea` and `preb` would run. I tried to separate `pretask`, `task` and `posttask` when it made sense, but maybe a super-long `task` command is clearer. I don't think anyone is running `preXXX` explicitly in the command line.

Reply:
thanks @DanReyLop - I was mainly asking if the `pre…` and `post…` scripts are run automatically because there were some I looked at and couldn't figure out how or when they run, for example, the `pretest` set